### PR TITLE
Student cv

### PR DESCRIPTION
**Pull request by Mariya-Sharibchenko**

### CODE REVIEW

File: src/atoms/DetailedFieldsForStudentCV/ContactsInfoForStudentCV.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  position: relative;
+  width: 36px;
+  height: 36px;
+  border-radius: 5px;
+  background-color: ${mainColor};
+  
+  &:after {
+    position: absolute;
+    content: ${({name}) => {
+      switch (name) {
```
Comment (Karador) [inline src/atoms/DetailedFieldsForStudentCV/ContactsInfoForStudentCV.tsx:47]: Better use enum

File: src/atoms/DetailedFieldsForStudentCV/EducationInfoForStudentCV.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { AdditionalEducationTitle, EducationLanguageTitle, IStudentEducationInfoProps } from 'context';
+
+import { StyledInfoItem, StyledInfoWrapper, StyledTitle, StyledText } from './styled';
+
+export const EducationInfoForStudentCV: React.FC<IStudentEducationInfoProps> = ({formal, additional, english}) => {
+  const { level, detailedInfo } = formal;
+
+  return (
+    <StyledInfoWrapper>
+      <StyledInfoItem>
```
Comment (Karador) [inline src/atoms/DetailedFieldsForStudentCV/EducationInfoForStudentCV.tsx:12]: Better use FC(title, text)

File: src/atoms/FieldsForStudentCV/ContactsFieldForStudentCV.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+export const ContactsFieldForStudentCV: React.FC<IContactsFieldForStudentCVProps> = ({title, isShowed, contacts}) => {
+  const { city, tel, eMail, socialMedia} = contacts;
+
+  return (
+    <WrapperFieldForStudentCV>
+      <StudentCVInfoTitle>{title}</StudentCVInfoTitle>
+
+      <StudentCVInfoContent>
+        { isShowed
+          ? <ContactsInfoForStudentCV city={city}
```
Comment (Karador) [inline src/atoms/FieldsForStudentCV/ContactsFieldForStudentCV.tsx:23]: Component
   props
/>

**Review by Karador — APPROVED** [review]

File: src/atoms/BookmarkButton/BookmarkButton.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import bookmarkWhite from 'public/bookmark-icon_white.svg';
+import bookmarkColored from 'public/bookmark-icon_colored.svg';
+
+const { mainColor, secondaryColor, mainText } = Colors;
+
+export interface IStyledBookmarkButton {
+  isMarked: boolean,
+}
+
+export interface IBookmarkButtonProps extends IStyledBookmarkButton{
```
Comment (suicaed) [inline src/atoms/BookmarkButton/BookmarkButton.tsx:15]: `IStyledBookmarkButton{` -> `IStyledBookmarkButton {`

File: src/atoms/BookmarkButton/BookmarkButton.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  onClick?: () => void;
+
```
Comment (suicaed) [inline src/atoms/BookmarkButton/BookmarkButton.tsx:17]: Should we have empty string here?

File: src/atoms/BookmarkButton/BookmarkButton.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+DefaultBookmarkButton.args = {
+  isMarked: false
+};
+
+export const MarkedBookmarkButton = Template.bind({});
+
+MarkedBookmarkButton.args = {
+  isMarked: true
+};
```
Comment (suicaed) [inline src/atoms/BookmarkButton/BookmarkButton.stories.tsx:23]: It is better to add eslint rule 'newline at end of file'.

File: src/atoms/DetailedFieldsForStudentCV/EducationInfoForStudentCV.tsx [hunk 7]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { AdditionalEducationTitle, EducationLanguageTitle, IStudentEducationInfoProps } from 'context';
+
+import { StyledInfoItem, StyledInfoWrapper, StyledTitle, StyledText } from './styled';
+
+interface ITitleTextFieldProps {
+	title: string,
```
Comment (suicaed) [inline src/atoms/DetailedFieldsForStudentCV/EducationInfoForStudentCV.tsx:8]: So what is the tab size? Two spaces or four? It must be the same across all project.

**Review by suicaed — COMMENTED** [review]
Firstly, there must be common rules for code style. Look comments.

**Review by suicaed — APPROVED** [review]
Well done!

**Pull request merged**